dbux-projects/src/dataLib/progressLog.js

```diff
@@ -0,0 +1,52 @@
+
+import { newLogger } from 'dbux-common/src/log/logger';
+import bugResultHandler from './BugResult';
+import testRunHandler from './TestRun';
+
+const logger = newLogger('ProgressLog');
+const { log, debug, warn, error: logError } = logger;
+
+function newProgressLog() {
+  return {
+    testRuns: [],
+    bugResults: [],
+  };
+}
+
+async function processBugResult(obj, bug, result) {
```

Comment:
the name `obj` is ambiguous. What kind of object is it? Is there a better name to choose?